### DISCUSSION TITLE
[1.4] Input.Acceleration Iphone 에서 안돼요

### DISCUSSION THREAD

**Discussion by Deytooh** [naverz/zepeto-studio-kor#1002] [5. Q&A]
안녕하세요 ~~
Input.Acceleration 쓰고있는데, Android 에서 잘 돼고있는데 Iphone에서 안돼요... Input.acceleration 는 항상 0.0,0.0,0.0 ㅠㅠ 

1.5.1에서도 안돼요..다른 방법 없을까요 ?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 오류 상황에 대한 보고는 사용하신 코드와 함께 이슈 리포트를 통해 부탁 드립니다.
https://github.com/naverz/zepeto-studio-kor/issues
감사합니다.

**Reply by Deytooh** [reply]
@Zepeto-TechEvangelist 네 ! 해보겠습니다 ! 그러면 혹시 Accelerometer 쓰는 방법이 없나요 ? 다른 월드에서 어떤 사람이 쓰고있는데 방법이 있겠죠 ㅠㅠ
